show message queued date in localtime according to user preferrence

as it previously displayed it just in GMT without stating the timezone, which
is rather confusing

--- ../include/class.mail_queue.php~	2005-04-13 12:21:56.255342541 +0300
+++ ../include/class.mail_queue.php	2005-04-13 12:22:05.414836030 +0300
@@ -364,6 +364,7 @@
         if (count($res) > 0) {
             for ($i = 0; $i < count($res); $i++) {
                 $res[$i]['maq_recipient'] = Mime_Helper::decodeAddress($res[$i]['maq_recipient']);
+                $res[$i]['maq_queued_date'] = Date_API::getFormattedDate(Date_API::getUnixTimestamp($res[$i]['maq_queued_date'], 'GMT'));
             }
         }
         return $res;
